tests/e2e/core/fixtures/test-data.fixture.ts

```
@@ -0,0 +1,169 @@
+      allowedModels: [],
+      keyIds: [],
+      ...overrides,
+    }
+  }
+}
+
+/**
+ * Extended test with test data fixture
+ */
+export const testWithData = base.extend<TestDataFixtures>({
+  testData: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)